### PR TITLE
Move cli-reference to cli-usage

### PR DESCRIPTION
**Pull request by brendongo**

### CODE REVIEW

**Review by pabloest — APPROVED** [review]

File: docs/cli-usage.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# CLI Usage
+# CLI Reference
```
Comment (chmccreery) [inline docs/cli-usage.md:1]: Did you mean to change this? Title of PR makes me think you wanted "CLI Usage" not "CLI Reference"

Comment (pabloest) [reply docs/cli-usage.md:1]: After @brendongo’s PR #105 merged, he and I both realized there was already a doc page with CLI info (mostly exit codes). So we decided to combine the contents of Brendon’s PR with the CLI content that was already there.

Comment (chmccreery) [reply docs/cli-usage.md:1]: So should this be closed?

Comment (brendongo) [reply docs/cli-usage.md:1]: This PR is to merge the two pages under this new name. I felt reference was better for having both the --help and exit codes

**Pull request merged**